### ISSUE TITLE
How to use torch::where?

### ISSUE THREAD

**Issue by laizewei** [pytorch/pytorch#38542]
libtorch 1.5
How to use torch::where?  for emample:
-------------------------------------------------
import torch
import numpy as np
cc=np.array(range(0,24)).reshape(-1,4)
validIndex=np.where( ((cc[:,:2]>=0) & (cc[:,2:]>(1,2))).all(axis=1) )[0]
print(validIndex)
>>[0 1 2 3 4 5]
---------------------------------------------------
torch::Tensor cc = torch::range(0, 23, 1).view({ 4,6 }).view({-1,4});
	torch::Tensor c = cc.index({ torch::indexing::Slice(),torch::indexing::Slice(2,torch::indexing::None) });
?????

**Comment by jerryzh168** [comment]
Please use https://discuss.pytorch.org/ for questions
